frontend/web/components/pages/OrganisationSettingsPage.js

```diff
@@ -1201,9 +1197,12 @@ const OrganisationSettingsPage = class extends Component {
                               Audit webhooks let you know when audit logs occur,
                               you can configure 1 or more audit webhooks per
                               organisation.{' '}
-                              <ButtonLink href='https://docs.flagsmith.com/advanced-use/system-administration#audit-log-webhooks/'>
+                              <Button
+                                theme='text'
+                                href='https://docs.flagsmith.com/advanced-use/system-administration#audit-log-webhooks/'
```

Comment:
@dabeeeenster I have a feeling this has changed / will change?

Reply:
This seems a bit unrelated to this PR, though can update if the above gets merged soon.